xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -5767,6 +5767,68 @@ TEST_P(ParameterizedFp8GemmRewriteTest,
       )");
 }
 
+TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABUnscaledDWithAllGatherF8) {
+#if CUDA_VERSION < 12000
+  GTEST_SKIP() << "A matrix bias on a matmul is only supported in CUDA 12";
```

Comment:
Adding all-gather to the list of ops that can be exchanged with dequantization isn't really tied to the presence of a bias. You can use the same text as in the other FP8 tests.